packages/story-editor/src/components/mediaRecording/settingsModal.js

```diff
@@ -107,6 +125,10 @@ function SettingsModal() {
   const audioInputs = mediaDevices
     .filter((device) => device.kind === 'audioinput')
     .map(({ deviceId, label }) => ({ value: deviceId, label }));
+  const videoEffects = [
+    { value: 'none', label: __('None', 'web-stories') },
+    { value: 'blur', label: __('Background blur', 'web-stories') },
```

Comment:
<!-- trivial preference-based requests -->
**🥜 nitpick:**
If we'll add other effects in the future, might be nice to create a constant for it. Could already now be `VIDEO_EFFECTS: { NONE: 'none', BLUR: 'blur' }` since `VIDEO.EFFECTS.BLUR` is already in multiple files.